Gpg key feedback resolution (#623)
ubuntu/ubuntu-project-docs: docs/contributors/setup/set-up-and-manage-pgp-keys.md

```

Finally, store these backups **offline** and **encrypted** and remove them from your system.
We acknowledge that this leaves the details of that backup setup intentionally under-specified.

> Having any setup that is truly offline and encrypted will in most places be better than what

Reviewer: ```suggestion
Having *any* truly offline and encrypted setup will, in most places, be better than what
```

has been used before.

> Here variety is ok and there likely is not only "one right setup", over time

Reviewer: ```suggestion
Here variety is ok and there is more than "one right setup". Over time
```
